### PR TITLE
Fixes #10830 - Humanized state description for Pulp and Candlepin async actions

### PR DESCRIPTION
**Pull request by iNecas**
To be displayed in task details and Dynflow console. Requires 
https://github.com/theforeman/foreman-tasks/pull/111 and 
https://github.com/Dynflow/dynflow/pull/153

### CODE REVIEW

File: app/lib/actions/candlepin/abstract_async_task.rb [hunk 1]
```diff
@@ -10,6 +10,21 @@ def run(event = nil)
         end
       end
 
+      def humanized_state
+        case state
+        when :running
+          if self.external_task.nil?
+            _("initiating Candlepin task")
+          else
+            _("checking Candlpin task status")
```
Comment (thomasmckay) [inline app/lib/actions/candlepin/abstract_async_task.rb]: typo Candlepin

Comment (iNecas) [inline app/lib/actions/candlepin/abstract_async_task.rb]: good catch! updated

**Comment by iNecas** [comment]
@thomasmckay ok to merge?

**Comment by thomasmckay** [comment]
@iNecas No other changes are required for all places in UI, both foreman tasks and katello, to render this?
Running with https://github.com/theforeman/foreman-tasks/pull/111 I get errors viewing a repo sync task but will try again on a fresh system.

**Comment by iNecas** [comment]
You need https://github.com/theforeman/foreman-tasks/pull/111 and https://github.com/Dynflow/dynflow/pull/153, what error are you seeing?

**Comment by thomasmckay** [comment]
ack

**Pull request merged**